### DISCUSSION TITLE
图片广场或相册，图片详情内查看优化建议

### DISCUSSION THREAD

**Discussion by YoweTeam** [lsky-org/lsky-pro#700] [想法]
最近购买了企业版，上传了很多图片；使用过程中，发现想翻到下一张图片；还需要自己到上一页，重新点下一张图片，需要求记住刚使用的图片，是在什么位置，才能正确选择下一张或上一张，使用体验有点拉胯。
建议：
   1.图片详情：增加下一张与上一张
   2.涉及范围：图片广场、固定的相册

有时候，上传的图片多了的情况下，想找一些相似的图片，没有便捷的方式，可以适当增加本站推荐的选项，在图片详情内。

**Comment by 0xxb** [comment]
这个是刻意为之的，图片广场使用的是游标分页的查询方式，游标查询可以在数据量百万甚至千万数据量的情况下可以做到没有压力的分页，弊端就是你说的这个问题。

下个版本我看这块如何优化一下，或许可以增加个开关控制是否需要使用游标查询。

**Reply by YoweTeam** [reply]
那真的是太棒了。
